cmake/OpenCVFindVA.cmake

```diff
@@ -0,0 +1,19 @@
+# Main variables:
+# HAVE_VA for conditional compilation OpenCV with/without libva
+
+if(UNIX AND NOT ANDROID)
+    find_path(
+    VA_INCLUDE_DIR
+    NAMES va/va.h
+    PATHS "/usr/include"
+    PATH_SUFFIXES include
+    DOC "Path to libva headers")
+endif()
+
+if(VA_INCLUDE_DIR)
+    set(HAVE_VA TRUE)
+    set(VA_EXTRA_LIBS "-lva")
```

Comment:
Most part of CMake modules uses these variables names:
- `${LIB}_INCLUDE_DIRS`
- `${LIB}_LIBRARIES`